day5/.solutions/02-GT4Py-concepts.ipynb

```diff
@@ -16,22 +16,9 @@
   },
```

Comment:
<p>GT4Py is a domain-specific language (DSL) to express stencil computations as regular Python functions. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ofuhrer/HPC4WC/pull/88/discussion/#comment-2168570052'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>